ibis/backends/pyspark/tests/conftest.py

```diff
@@ -38,7 +38,7 @@ def client():
     from pyspark.sql import SparkSession
 
     session = SparkSession.builder.getOrCreate()
-    client = ibis.pyspark.connect(session)
+    client = ibis.backends.pyspark.connect(session)
```

Comment:
Not sure why this isn't caught before. I manually ran the tests locally to verify this.